CMakeLists.txt

```diff
@@ -37,10 +37,20 @@ exec_program(${PYTHON_CONFIG}
   ARGS "--includes"
   OUTPUT_VARIABLE PYTHON_CFLAGS)
 
+execute_process(
+    COMMAND ${PYTHON} -c "import numpy; print(numpy.get_include())"
+    RESULT_VARIABLE NUMPY_INCLUDE_STATUS
+    OUTPUT_VARIABLE NUMPY_INCLUDE)
+
+if("${NUMPY_INCLUDE}" STREQUAL "")
```

Comment:
You may also want to check that `${NUMPY_INCLUDE_STATUS}` equals zero.

Reply:
Good idea. I changed to only that actually, since that should be enough.

Reply:
Actually, I went ahead and used both. Even though an empty string is technically a valid path, I don't think we ever want to see that for the numpy include path.